java/pubsub/src/main/java/com/example/pubsub/CreatePullSubscriptionExample.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.pubsub;
```

Comment:
s/com.example/io.opencensus.exemple
Same for other files.

Reply:
In the README I mention that this code is a modification of the Pub/Sub example in https://github.com/GoogleCloudPlatform/java-docs-samples.

I thought about changing the class path as suggested but kept the same so that the comments and instructions from the original example apply here, which simplifies things for the user.

I'm open to changing it if that's really better - thoughts?

Reply:
Make sense, in this case I would prefer to stay the same with the original Pub/Sub example.